test/services/videoconference/index.js

```diff
@@ -1,288 +0,0 @@
-/* eslint-disable no-unused-expressions */
```

Comment:
why were the 3 tests removed - how is this related to the current PR?

Reply:
because locally the tests are not running with these. I forgot to remove the tests. There is no videoconference in feathers.

Reply:
not a big thing, but would be to separate them to avoid confusion

Reply:
i removed them, because i needed to run tests locally and they were not working with these tests. It is another topic, true, but as this change does not affect any functionality and this PR is small enough, that this 3 deleted files do not confuse too much, i would really like to ease the process of removing those files :)